### PR TITLE
add HTML for header

### PR DESCRIPTION
**Pull request by ArthurSmid**
This is the header image as HTML and CSS.

### CODE REVIEW

File: EugeneFoodScene/Client/Shared/Header.razor [hunk 1]
```diff
@@ -1,4 +1,19 @@
 ﻿<div class="flex-row header justify-content-center">
     <SearchBar />
-    <RadzenImage Path="images/EugeneRestaurants-GoogleHeader.png" />
-</div>
\ No newline at end of file
+    
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@800&display=swap" rel="stylesheet">
```
Comment (markdav-is) [inline EugeneFoodScene/Client/Shared/Header.razor]: these belong in index.html in wwwroot

File: EugeneFoodScene/Client/Shared/Header.razor [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
-</div>
\ No newline at end of file
+    
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@800&display=swap" rel="stylesheet"> 
+<link href="https://fonts.googleapis.com/css2?family=Krona+One&display=swap" rel="stylesheet">
+
+<div class="title-container">
+  <div class="top-row">
+    <span class="top-line left-line"> \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\</span> 
+    <h2 class="support">Support Local Restaurants</h2>
```
Comment (markdav-is) [inline EugeneFoodScene/Client/Shared/Header.razor]: add a clls for the h123-reset

**Review by markdav-is — CHANGES_REQUESTED** [review]

File: EugeneFoodScene/Client/Shared/Header.razor (hunk 2 above)
Comment (markdav-is) [reply EugeneFoodScene/Client/Shared/Header.razor]: add class is what I meant

**Review by ArthurSmid — COMMENTED** [review]
removed the Google Font links

**Review by markdav-is — APPROVED** [review]

**Pull request merged**